NGitLab/Models/Variable.cs

```diff
@@ -19,6 +27,17 @@ public class Variable
     [JsonPropertyName("masked")]
     public bool Masked { get; set; }
 
+    [JsonPropertyName("raw")]
+    public bool Raw { get; set; }
+
+    /// <summary>
+    /// The environment scope of a variable
+    /// </summary>
+    /// <returns>The environment scope of a variable</returns>
+    /// <remarks>
+    /// Create and Update of project variable: All tiers: Free, Premium, Ultimate <br/>
+    /// Create and Update of group variable: Premium and Ultimate only.
+    /// </remarks>
     [JsonPropertyName("environment_scope")]
     public string Scope { get; set; }
```

Comment:
```suggestion
    public string EnvironmentScope { get; set; }
```

I think we could let the full name, as it's not the scope of the variable in itself, but linked to environments

Reply:
Resolved with bf58a1c80cc5c00ba58a3519337f04dcee412149